### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vladislavaSim**
Capycalendar v1.0 is here. To use this, you should pass year and month number to the function and it will return number of days.

### CODE REVIEW

File: README.md
Comment (serhiihiba) [inline README.md:1]: Where's api?

File: package.json
Comment (serhiihiba) [inline package.json:1]: ???

File: index.js
Comment (serhiihiba) [inline index.js:1]: index.js should only export, no main code

**Review by serhiihiba — CHANGES_REQUESTED** [review]
Where's bundle? And you can't push major version in first PR lol

File: index.js
Comment (vladislavaSim) [reply index.js:1]: Ive taken your comments into account and opened new PR. Thank you!

**Comment by vladislavaSim** [comment]
@serhiijun Ive taken your comments into account and opened new PR. Thank you! 😎

**Comment by vladislavaSim** [comment]
@serhiijun Ive taken your comments into account and opened new PR. Thank you!

**Pull request closed without merging**